hurumap/static/js/charts.js

```diff
@@ -14,6 +14,31 @@ Chart({
 function Chart(options) {
   var chart = {};
 
+    default_colorbrewer = {
```

Comment:
`defaultColorbrewer`, JS naming.

Reply:
> Looks good @phillipahereza and we could be on our way here...
> 
> One thing missing is `chartHeight`. Please include it in the configuration.

@kilemensi won't this change the height of all the charts, is that the what we're trying to achieve

Reply:
Yes @phillipahereza, that's by design: https://github.com/CodeForAfrica/HURUmap/blob/377dc7a46770145081142bf21e2ac4c6e80f426a/hurumap/templates/profile/profile_detail.html#L969